example/menu/gui.gui

```diff
@@ -0,0 +1,248 @@
+script: "/example/menu/gui.gui_script"
```

Comment:
The two text nodes have scale applied which makes them bigger than the underlying box. gui.pick_node() will then pick both buttons at the same time and load both collections (see comment about elseif below)